### PR TITLE
feat: add maangchi

### PR DESCRIPTION
**Pull request by wengtad**
One of the website in #489

### CODE REVIEW

File: recipe_scrapers/maangchi.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        after = self.soup.find("h4", string="Ingredients:").find_all_next("li")
+        lstBfr = [normalize_string(b.get_text()) for b in before]
+        lstAfr = [normalize_string(a.get_text()) for a in after]
+        lst = []
+
+        # iterate to reverse the list
+        for i in lstBfr:
+            # reversing the list
+            lst.insert(0, i)
+        lstBfr = lst
```
Comment (jayaddison) [inline recipe_scrapers/maangchi.py:38]: (optional) a slightly easier way to do this:

```suggestion
        lstBfr.reverse()
```

**Review by jayaddison — APPROVED** [review]
Looks good to me :+1:

File: recipe_scrapers/maangchi.py (hunk 1 above)
Comment (wengtad) [reply recipe_scrapers/maangchi.py]: big help, thanks 👍

Comment (wengtad) [reply recipe_scrapers/maangchi.py]: could I get the `hacktoberfest-accepted` labels please, thanks

Comment (jayaddison) [reply recipe_scrapers/maangchi.py]: Yep!  Labels added.

**Pull request merged**